app/uk/gov/hmrc/gform/lookup/LookupOptions.scala

```diff
@@ -35,11 +35,15 @@ case class LookupOptions(options: Map[LookupLabel, LookupInfo]) extends AnyVal {
       }
       .map(_._1)
 
-  def sortLookupByPriorityAndLabel: List[LookupLabel] =
+  def sortLookupByPriorityAndLabel(priorityType: Option[Priority]): List[LookupLabel] =
     options.toList
       .sortBy {
-        case (label, DefaultLookupInfo(_, _))                        => (LookupPriority(1), label)
-        case (label, CountryLookupInfo(_, _, _, priority, _, _, _))  => (priority, label)
+        case (label, DefaultLookupInfo(_, _)) => (LookupPriority(1), label)
+        case (label, CountryLookupInfo(_, _, _, priority, priorityUk, _, _, _)) =>
+          priorityType match {
+            case Some(priorityType) => if (priorityType == Uk) (priorityUk, label) else (priority, label)
```

Comment:
Can we use `===` instead of `==` for type-safe equality?

Reply:
Updated